guides/akka-cluster-kubernetes-k8s-deploy/akka-cluster-example/build.sbt

```diff
@@ -0,0 +1,74 @@
+import sbt._
+import com.typesafe.sbt.SbtScalariform.ScalariformKeys
+import de.heikoseeberger.sbtheader.HeaderPattern
+import scalariform.formatter.preferences.AlignSingleLineCaseStatements
+import com.typesafe.sbt.packager.docker._
+
+lazy val akkaVersion = "2.5.4"
+
+name := "myapp"
+
+scalaVersion := "2.11.11"
```

Comment:
2.12.3 possible?

Reply:
Yes, it's possible indeed - changed & retested. Updated now to `2.12.3`.